okex-client/src/client/mod.rs

```diff
@@ -34,13 +34,40 @@ pub struct OkexClient {
 }
 
 impl OkexClient {
-    pub fn new(config: OkexClientConfig) -> Self {
+    fn new(config: OkexClientConfig) -> Self {
         Self {
             client: ReqwestClient::new(),
             config,
         }
     }
 
+    pub async fn create(config: OkexClientConfig) -> Result<Self, OkexClientError> {
+        // 1. Get account configuration
+        let client = Self::new(config);
```

Comment:
There is no need for `new` and `create`. Just rename `create` to `new` and inline what is now `new` here